src/main/java/net/juniper/netconf/Device.java

```diff
@@ -88,6 +89,8 @@ public Device(
             @NonNull String hostName,
             Integer port,
             Integer timeout,
```

Comment:
Is this still required? Where is this set?

Reply:
I think it is down  here in NetconfSession:

 NetconfSession(Channel netconfChannel, int connectionTimeout, int commandTimeout,
If only timeout is set, then Vlad's change sets both timeout flavors to the value of timeout.

Reply:
yes, it's for compatibility with old API